README.md

```diff
@@ -70,7 +70,7 @@ When developing this library, please use `make` (or `make … BUILDTAGS=…`) to
 
 ## License
 
-Apache License 2.0
+[Apache License 2.0](https://github.com/containers/image)
```

Comment:
I'm not sure the containers/image GitHub project is where you want to link to for this...
Plus there's some kind of test issue too.

Reply:
okay if you are ref [Apache License 2.0] then why is the licence url not attached ?

Reply:
Rather than the top level repository, I was thinking something like:
```suggestion
[Apache License 2.0](https://github.com/containers/image/blob/master/LICENSE)
```

Reply:
Could you change the link as suggest by Tom?